alacritty.yml

```diff
@@ -452,18 +459,32 @@
   # If this is `true`, the cursor is temporarily hidden when typing.
   #hide_when_typing: false
 
-# Regex hints
+# Hints
 #
-# Terminal hints can be used to find text in the visible part of the terminal
-# and pipe it to other applications.
+# Terminal hints can be used to find text or hyperlinks in the visible part of
+# the terminal and pipe it to other applications.
 #hints:
   # Keys used for the hint labels.
   #alphabet: "jfkdls;ahgurieowpq"
 
   # List with all available hints
   #
-  # Each hint must have a `regex` and either an `action` or a `command` field.
-  # The fields `mouse`, `binding` and `post_processing` are optional.
+  # Each hint must consist of:
+  # - At least one of a `regex` or a `hyperlink` field, specifying which the
+  #   hint applied on.
+  #
+  #   `regex` takes an Regular Expression as string for matching text.
+  #   `hyperlink` takes `true` to match hyperlink escape from inner program.
+  #   Note that target URL of a hyperlink may differ from its text displayed.
+  #
+  #   `hyperlink` takes precedence over `regex` when they both match on the
+  #   same text.
+  #
+  # - Either an `action` or a `command` field, specifying the action when
+  #   triggered.
+  #
+  # - Zero or more optional fields, including `mouse`, `binding` and
+  #   `post_processing`.
```

Comment:
This is too verbose, with this much text nobody will read through the hints documentation anymore. The existing documentation was already **a lot** and this almost doubles its size.

I also find it questionable if regex and hint should be combined in the same hint. It seems somewhat strange and clunky.

And if we decide to support multiple things in the same hint, should we also allow multiple regexes?

Reply:
> I also find it questionable if regex and hint should be combined in the same hint. 

If they are not, we need to use different key bindings for them, instead of Control-Shift-U to show both regex matched and hyperlink escaped text. I don't think it provide a good user experience.

Reply:
The docs are simplified.
But I keep hyperlink and regex NOT exclusive since I prefer navigate both with one key shortcut. Also users can still choose to use different ones if they want.

> should we also allow multiple regexes

No. Regex can already be naturally joined by `|`.